.travis.yml

```diff
@@ -1,36 +1,49 @@
-osx_image: xcode9.3
 language: objective-c
 
-cache:
-  directories:
-  - Carthage
-  - Xcode/Testing-iOS/Pods
-  - Xcode/Testing-macOS/Pods
+jobs:
+  include:
+    - stage: Swift PM
+      osx_image: xcode12.2
+      script:
+      # Remove the Xcode project so `xcodebuild` would be able to test the Swift PM project.
+      - rm -rf *.xcodeproj
+      # Test the Swift PM project using `xcodebuild`.
+      # The test is performed using `xcodebuild` rather than `swift test` due to this bug:
+      # https://bugs.swift.org/browse/SR-13560
```

Comment:
Great job on the workaround! Hopefully Apple fixes this bug someday. Is there a [Feedback Assistant](https://feedbackassistant.apple.com) report number that could be added to that ticket?

Reply:
Im not aware of any